src/components/Dialogs/BaseDialog/BaseDialog.style.ts

```diff
@@ -12,13 +12,17 @@ export const StyledContainer = styled.div`
   width: 90%;
   max-width: 440px;
   min-height: 150px;
-  overflow: hidden;
+  max-height: 550px;
+  overflow-y: scroll;
   margin: ${sizes(16)} auto;
   color: ${colors.white};
   background-color: ${colors.gray[700]};
   padding: var(--dialog-padding);
   box-shadow: 0 8px 8px rgba(0, 0, 0, 0.12), 0 24px 40px rgba(0, 0, 0, 0.16);
-
+  ${media.small} {
+    max-height: 650px;
+    overflow: hidden;
```

Comment:
Not 100% sure, but with `overflow: auto` we could get rid of the `overflow: hidden` entirely, and just adjust `max-height`. Hidden overflow could be bad for laptop screens. But it needs some testing.

Also maybe we should consider setting these values based on the height of the viewport, not width.